pems-server/src/main/java/org/maroubra/pemsserver/monitoring/utsapi/WebSensor.java

```diff
@@ -0,0 +1,80 @@
+package org.maroubra.pemsserver.monitoring.utsapi;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import retrofit2.Call;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+import java.io.*;
+import java.time.LocalDateTime;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class WebSensor {
+    private LocalDateTime fromDate;
+    private LocalDateTime toDate;
+    private int pollIntervalMinutes = 60;
+    private String family;
+    private String subSensor;
+    private String sensor;
+    private String apiLink = "http://eif-research.feit.uts.edu.au/api/";
+    private static Logger log = LoggerFactory.getLogger(WebSensor.class);
+
+    private Retrofit retrofit = new Retrofit.Builder()
+            .baseUrl(apiLink)
+            .addConverterFactory(GsonConverterFactory.create())
+            .build();
+
+    private WebApiRequest service = retrofit.create(WebApiRequest.class);
+
+    public WebSensor(String family, String sensor, String subSensor) {
+        this.family = family;
+        this.subSensor = subSensor;
+        this.sensor = sensor;
+    }
+
+    public void setPollIntervalMinutes(int pollIntervalMinutes) {
+        this.pollIntervalMinutes = pollIntervalMinutes;
+    }
+
+    public void setDatesPollInterval() {
+        fromDate = LocalDateTime.now().minusMinutes(pollIntervalMinutes).withNano(0);
```

Comment:
Following my comments on `WebApiRequest`, what do you think about using the `hc` endpoint. I've found its always good to deal with UTC date times in systems and then convert it to the local date time in the frontend for display. It mostly just makes the system more consistent across usage.

Reply:
Sure it would definitely make sense to use the same date format in all of our sensor data. Changed the method called to hc but kept the readable json for testing/prototyping.